src/qiskit_iqm/fake_backends/iqm_fake_backend.py

```diff
@@ -0,0 +1,185 @@
+
+    def _create_noise_model(self) -> NoiseModel:
+        """
+        Compiles a noise model from attributes of this instance of IQMSimulator.
+        """
+        noise_model = NoiseModel(basis_gates=self.basis_gates)
+
+        # Add single-qubit gate errors to noise model
+        for gate in self.basis_1_qubit_gates:
+            for i in range(self.chip_sample.number_of_qubits):
+                thermal_relaxation_channel = thermal_relaxation_error(
+                    self.chip_sample.t1s[i], self.chip_sample.t2s[i], self.chip_sample.one_qubit_gate_durations[gate]
+                )
+                depolarizing_channel = depolarizing_error(
+                    self.chip_sample.one_qubit_gate_depolarization_rates[gate][i], 1
+                )
+                full_error_channel = thermal_relaxation_channel.compose(depolarizing_channel)
+                noise_model.add_quantum_error(full_error_channel, gate, [i])
+
+        # Add two-qubit gate errors to noise model
+        for gate in self.basis_2_qubit_gates:
+            for connection in list(self.chip_sample.two_qubit_gate_depolarization_rates[gate].keys()):
+                first_qubit = connection[0]
+                second_qubit = connection[1]
+
+                thermal_relaxation_channel = thermal_relaxation_error(
+                    self.chip_sample.t1s[first_qubit],
+                    self.chip_sample.t2s[first_qubit],
+                    self.chip_sample.two_qubit_gate_durations[gate],
+                ).tensor(
+                    thermal_relaxation_error(
+                        self.chip_sample.t1s[second_qubit],
+                        self.chip_sample.t2s[second_qubit],
+                        self.chip_sample.two_qubit_gate_durations[gate],
+                    )
+                )
+                depolarizing_channel = depolarizing_error(
+                    self.chip_sample.two_qubit_gate_depolarization_rates[gate][connection], 2
+                )
+                full_error_channel = thermal_relaxation_channel.compose(depolarizing_channel)
+                noise_model.add_quantum_error(full_error_channel, gate, [first_qubit, second_qubit])
+
+        return noise_model
+
+    def run(self, run_input: Union[QuantumCircuit, list[QuantumCircuit]], **options) -> JobV1:
+        """
+        Run `run_input` on the fake backend using a simulator.
+
+        This method runs circuit jobs (an individual or a list of QuantumCircuit
+        ) and returns a :class:`~qiskit.providers.JobV1` object.
+
+        It will run the simulation with a noise model of the fake backend (e.g. Adonis).
+
+        Args:
+            run_input (QuantumCircuit or list of QuantumCircuit): An
+                individual or a list of
+                :class:`~qiskit.circuits.QuantumCircuit` objects to run on the backend.
+            options: Any kwarg options to pass to the backend.
+        Returns:
+            JobV1: The job object for the run
```

Comment:
Here, too, the type is unnecessary.
```suggestion
            The job object representing the run.
```